File: src/tool/spring-boot-generator/output/spider/db/taskMainConfig.js
let taskMainConfig = {
	name: "taskMainConfig",
	description: "任务主配置",
	columns: {
		id: {
			name: "id",
			description: "编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		taskId: {
			name: "taskId",
			description: "任务id",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		pattern: {
			name: "pattern",
			description: "pattern",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		encoding: {
			name: "encoding",
			description: "编码",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		agent: {
			name: "agent",
			description: "浏览器代理",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		timeout: {
			name: "timeout",
			description: "超时",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		scriptVersion: {
			name: "scriptVersion",
			description: "脚本版本",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		}
	}
};

let columnsArray = [
	"id",
	"taskId",
	"pattern",
	"encoding",
	"agent",
	"timeout",
	"scriptVersion"
];

function create() {
  return {};
}

taskMainConfig.columnsArray=columnsArray;
taskMainConfig.create=create;
module.exports=taskMainConfig;

File: src/tool/spring-boot-generator/output/spider/spider/src/main/java/com/spider/pojo/entity/TaskMainConfig.java
package com.spider.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;



@ApiModel("任务主配置")
@Getter
@Setter
public class TaskMainConfig  {

    @ApiModelProperty("编号")
    private Integer id;

    @ApiModelProperty("任务id")
    private Integer taskId;

    @ApiModelProperty("pattern")
    private String pattern;

    @ApiModelProperty("编码")
    private String encoding;

    @ApiModelProperty("浏览器代理")
    private String agent;

    @ApiModelProperty("超时")
    private Integer timeout;

    @ApiModelProperty("脚本版本")
    private String scriptVersion;

}